cmd/rest/client.go

```diff
@@ -67,6 +68,13 @@ func (c *Client) Call(method string, values url.Values, body io.Reader) (reply i
 	return resp.Body, nil
 }
 
+// CloseIdleConnections closes all idle connections of the underlying http client
+func (c *Client) CloseIdleConnections() {
```

Comment:
you can rename this function to a generic `Close()` as there is no need for the rest client to expose the details of what it is doing to higher layers. You can rename the struct member `httpCloser` from a generic name to a more specific name "closeIdleConnections()` as this is an internal detail in rest-client package and makes it more readable

Reply:
I agree we can rename the function simply as Close() - http functions doesn't need to be exposed or we don't need bring their variants here.